test/gtest/ucs/test_conn_match.cc

```diff
@@ -0,0 +1,276 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include <common/test.h>
+
+#include <ucs/datastruct/conn_match.h>
+#include <ucs/sys/sys.h>
+
+#include <vector>
+
+class test_conn_match : public ucs::test {
+public:
+    typedef struct {
+        ucs_conn_match_elem_t conn_match;
+        bool                  is_exp;
```

Comment:
maybe keep `ucs_conn_match_queue_type_t` instead of `bool`?
Then you will not need type translations and `test_conn_match::m_queue_type`

Reply:
done